Microsoft.Toolkit.Uwp.SampleApp/SamplePages/PrintHelper/PrintHelperPage.xaml

```diff
@@ -25,54 +47,81 @@
-                               Text="It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum."
-                               TextWrapping="Wrap" />
-                    <TextBlock FontSize="14"
-                               FontStyle="Italic"
-                               Text="From http://www.lipsum.com/"
-                               TextWrapping="Wrap" />
-                </StackPanel>
+                <ListView x:Name="PrintableContent"
+                          ItemsSource="{x:Bind PrintSampleItems}"
+                          SelectionMode="None"
+                          Margin="20">
+                    <ListView.ItemTemplate>
+                        <DataTemplate>
+                            <Grid>
+                                <Grid.RowDefinitions>
+                                    <RowDefinition Height="auto" />
+                                    <RowDefinition Height="auto" />
+                                </Grid.RowDefinitions>
+                                <Grid.ColumnDefinitions>
+                                    <ColumnDefinition Width="auto" />
+                                    <ColumnDefinition Width="*" />
+                                </Grid.ColumnDefinitions>
+                                <Image Width="200"
+                                       Height="200"
+                                       Source="{Binding PicturePath}"
+                                       Stretch="UniformToFill"
+                                       VerticalAlignment="Top"
+                                       Margin="0 0 0 10"
+                                       Grid.RowSpan="2" />
+                                <TextBlock FontSize="20"
+                                           Text="{Binding Description}"
+                                           TextWrapping="Wrap"
+                                           Margin="10 0 0 0"
+                                           Grid.Column="1" />
+                                <TextBlock FontSize="14"
+                                           FontStyle="Italic"
+                                           Text="{Binding SourceUrl}"
+                                           TextWrapping="Wrap"
+                                           Margin="10 10 0 0"
+                                           Grid.Row="1"
+                                           Grid.Column="1" />
+                            </Grid>
+                        </DataTemplate>
+                    </ListView.ItemTemplate>
+                </ListView>
             </Grid>
             <StackPanel Grid.Row="1"
                         Margin="20"
                         HorizontalAlignment="Center"
                         Orientation="Horizontal">
                 <Button Click="Print_Click"
```

Comment:
Should we move these to `Shell.Current.RegisterNewCommand`?

Reply:
Yes, we should

Reply:
Great idea! I moved the older buttons too...